lib/datastore/transaction.js

```diff
@@ -418,10 +417,12 @@ Transaction.prototype.commit_ = function(callback) {
 
   // Take the `req` array built previously, and merge them into one request to
   // send as the final transactional commit.
-  var reqOpts = this.requests_.reduce(function(acc, req) {
-    return extend(true, acc, req);
-  }, {});
-
+  var reqOpts = {
+    mutations: this.requests_.reduce(function(acc, req) {
```

Comment:
Is there a module we can use for this (or just add it to util?) I find myself doing this for BigTable in places, as well.